docs/concepts/what-is-an-activity.md

```diff
@@ -26,10 +26,15 @@ In its simplest form, a Temporal Activity is a function or an object method in o
 Temporal does not recover Activity state in case of failures.
 Therefore an Activity function is allowed to contain any code without restrictions.
```

Comment:
Therefore, an Activity Definition has no restrictions on its contents. (and link to the page!)